submit_cgap/scripts/submit_ontology.py

```diff
@@ -41,8 +45,21 @@ def main(simulated_args_for_testing=None):
                 env=args.env,
                 validate_only=args.validate_only,
                 app=args.app,
-                submission_protocol=args.submission_protocol,
-        )
+                submission_protocol=args.submission_protocol)
+
+
+def verify_ontology_file(ontology_filename: str) -> bool:
+    if not os.path.exists(ontology_filename):
+        raise ScriptFailure(f"ERROR: Cannot find specified ontology (JSON) file: {ontology_filename}")
```

Comment:
Oops. I screwed this up. Should have said don't ever put words like `ERROR:` in an error message. Those are supposed to get added by the tool catching the error, since sometimes they might want to convert them to warnings or log messages.
```suggestion
        raise ScriptFailure(f"Cannot find specified ontology (JSON) file: {ontology_filename}")
```